net-misc/httpie/httpie-1.0.2.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
```

Comment:
looks like httpie requires python's `ssl` useflag.
you'll need 
`PYTHON_REQ_USE="ssl(+)"` before inherit line.

that `(+)` is needed because some python implementations support ssl unconditionally.
https://wiki.gentoo.org/wiki/Project:Python/distutils-r1#PYTHON_REQ_USE
https://wiki.gentoo.org/wiki/Project:Python/Implementation_USE_flags